### PR TITLE
Update CassandraContainer comment to reflect compaitiblity with 4.x

### LINKED ISSUE #7012: [Enhancement]: Cassandra 4 support
### Module

Cassandra

### Proposal

I noticed the cassandra container only supports 2.x and 3.x. I'd like to see support for Cassandra server 4.x
I would be willing to help with this. I have experience getting cassandra 4 to run: https://github.com/jsevellec/cassandra-unit/pull/325
but I am not familiar with testcontainer's code base.

### PR DESCRIPTION
**Pull request by wakingrufus**
update comment to reflect cassandra 4.x compatibility fixes #7012

### CODE REVIEW

**Comment by eddumelendez** [comment]
Can we add a proper test to make sure it works, please?

**Comment by wakingrufus** [comment]
> Can we add a proper test to make sure it works, please?

yep, I was actually already working on that before you commented :) I just pushed an update

File: modules/cassandra/src/test/java/org/testcontainers/containers/CassandraServer4Test.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+import com.datastax.oss.driver.api.core.CqlIdentifier;
+import com.datastax.oss.driver.api.core.CqlSession;
+import com.datastax.oss.driver.api.core.metadata.schema.KeyspaceMetadata;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CassandraDriver4Test {
```
Comment (eddumelendez) [inline modules/cassandra/src/test/java/org/testcontainers/containers/CassandraServer4Test.java:11]: filename and class name doesn't match.
```suggestion
public class CassandraServer4Test {
```

File: modules/cassandra/src/main/java/org/testcontainers/containers/CassandraContainer.java [hunk 2]
```diff
@@ -21,7 +21,7 @@
 /**
  * Cassandra container
  *
- * Supports 2.x and 3.x Cassandra versions
+ * Supports 2.x, 3.x, and 4.x Cassandra versions
```
Comment (eddumelendez) [inline modules/cassandra/src/main/java/org/testcontainers/containers/CassandraContainer.java:24]:
```suggestion
 *  Testcontainers implementation for Apache Cassandra.
```

File: modules/cassandra/src/main/java/org/testcontainers/containers/CassandraContainer.java [hunk 3]
```diff
@@ -21,7 +21,7 @@
 /**
  * Cassandra container
  *
- * Supports 2.x and 3.x Cassandra versions
+ *  Testcontainers implementation for Apache Cassandra.
```
Comment (eddumelendez) [inline modules/cassandra/src/main/java/org/testcontainers/containers/CassandraContainer.java:24]:
```suggestion
 * Testcontainers implementation for Apache Cassandra.
```

**Review by eddumelendez — APPROVED** [review]

**Comment by wakingrufus** [comment]
Thanks for the fixups!
Is there anything else we need? or just waiting for merge?

**Comment by eddumelendez** [comment]
Thank you for your contribution, @wakingrufus !

**Pull request merged**